routes/users.js

```diff
@@ -30,7 +30,7 @@ usersRouter.post('/register', async (req, res, next) => {
         } else if (password.length < 8) {
             next({message: 'Password must be a minimum of 8 characters long.'});
         } else {
-            const user = await createUser({username, password, firstName, lastName, email})
+            const user = await createUser(req.body)
```

Comment:
REQUIRED - Either here or in the `createUser` function, only pass the values we want to pass.  Otherwise someone could create a superUser with `isAdmin` on it.

Reply:
No more superUser for you